### ISSUE TITLE
System.Net.Http.Functional.Tests long running tests

### ISSUE THREAD

**Issue by ViktorHofer** [dotnet/runtime#68289]
Unsure if this issue is actionable but still wanted to file it to hear the area owners' opinions.

Configuration: `net7.0-Linux-Release-x64-CoreCLR_release-(Alpine.314.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20210910135833-1848e19`

Build: https://dev.azure.com/dnceng/public/_build/results?buildId=1719581

<details>

```
  Discovering: System.Net.Http.Functional.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.Net.Http.Functional.Tests (found 1224 of 1619 test cases)
  Starting:    System.Net.Http.Functional.Tests (parallel test collections = on, max threads = 2)
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClient_SelectedSites_Test.RetrieveSite_Succeeds [SKIP]
      Condition(s) not met: "IsSelectedSitesTestEnabled"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClient_SelectedSites_Test.RetrieveSite_Debug_Helper [SKIP]
      Condition(s) not met: "IsSelectedSitesTestEnabled"
    System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.Credentials_ServerUsesWindowsAuthentication_Success [SKIP]
      Condition(s) not met: "IsNtlmInstalled", "IsWindowsServerAvailable"
    System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.Credentials_ServerChallengesWithWindowsAuth_ClientSendsWindowsAuthHeader [SKIP]
      Condition(s) not met: "IsNtlmInstalled"
    System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.Proxy_DomainJoinedProxyServerUsesKerberos_Success [SKIP]
      Condition(s) not met: "IsDomainJoinedServerAvailable"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientEKUTest.HttpClient_ClientEKUServerAuth_Fails [SKIP]
      Condition(s) not met: "CanTestCertificates"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientEKUTest.HttpClient_NoEKUServerAuth_Ok [SKIP]
      Condition(s) not met: "CanTestCertificates"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientEKUTest.HttpClient_NoEKUClientAuth_Ok [SKIP]
      Condition(s) not met: "CanTestCertificates"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientEKUTest.HttpClient_ServerEKUClientAuth_Fails [SKIP]
      Condition(s) not met: "CanTestCertificates"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.Credentials_DomainJoinedServerUsesKerberos_UseIpAddressAndHostHeader_Success [SKIP]
      Condition(s) not met: "IsDomainJoinedServerAvailable"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.Proxy_DomainJoinedProxyServerUsesKerberos_Success [SKIP]
      Condition(s) not met: "IsDomainJoinedServerAvailable"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.Credentials_ServerChallengesWithWindowsAuth_ClientSendsWindowsAuthHeader [SKIP]
      Condition(s) not met: "IsNtlmInstalled"
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:02:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:02:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:02:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:02:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:02:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:02:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:02:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:02:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:02:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:02:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:02:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:02:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:02:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:02:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:02:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:02:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:02:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:02:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:02:07
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:04:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:04:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:04:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:04:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:04:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:04:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:04:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:04:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:04:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:04:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:04:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:04:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:04:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:04:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:04:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:04:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:04:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:04:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:04:07
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:06:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:06:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:06:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:06:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:06:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:06:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:06:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:06:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:06:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:06:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:06:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:06:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:06:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:06:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:06:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:06:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:06:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:06:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:06:07
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:08:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:08:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:08:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:08:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:08:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:08:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:08:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:08:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:08:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:08:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:08:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:08:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:08:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:08:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:08:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:08:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:08:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:08:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:08:07
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:10:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:10:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:10:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:10:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:10:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:10:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:10:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:10:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:10:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:10:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:10:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:10:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:10:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:10:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:10:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:10:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:10:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:10:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:10:07
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:12:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:12:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:12:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:12:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:12:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:12:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:12:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:12:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:12:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:12:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:12:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:12:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:12:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:12:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:12:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:12:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:12:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:12:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:12:07
   System.Net.Http.Functional.Tests: [Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http2.ConnectCallback_ConnectionPrefix_Success', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Http2.ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests.GetAsync_StatusLineNotFollowRFC_SuccessOnCore', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1CloseResponseStreamConformanceTests.ZeroByteRead_BlocksUntilDataAvailableOrNops', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:14:33
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_ConnectCallback_Http11.ConnectCallback_NoAlpn_OK', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Http2.ResetResponseStream_FrameReceived_Ignored', Elapsed: 00:14:30
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_PlaintextStreamFilter_Http11.PlaintextStreamFilter_ContextHasCorrectProperties_Success', Elapsed: 00:14:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Async.TestLoopbackAsync', Elapsed: 00:14:59
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandlerTest.SendAsync_No100ContinueReceived_RequestBodySentEventually', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandlerTest.SendAsync_1xxResponsesWithHeaders_InterimResponsesHeadersIgnored', Elapsed: 00:14:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_HttpClientHandlerTest_Headers_Http2.SendAsync_InvalidCharactersInHeader_Throw', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_Http1_TrailingHeaders_Test.GetAsync_ForbiddenTrailingHeaders_Ignores', Elapsed: 00:14:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http2.TestLoopbackAsync', Elapsed: 00:14:59
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpContentTest.ReadAsStringAsync_SetQuotedCharset_ParsesContent', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:14:28
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_AutoRedirect.AllowAutoRedirect_True_ValidateNewMethodUsedOnRedirection', Elapsed: 00:14:29
[Long Running Test] 'System.Net.Http.Functional.Tests.SocksProxyTest_Http1_Sync.TestLoopbackAsync', Elapsed: 00:14:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Decompression_Tests.DecompressedResponse_MethodSpecified_DecompressedContentReturned', Elapsed: 00:14:58
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates', Elapsed: 00:14:23
[Long Running Test] 'System.Net.Http.Functional.Tests.HttpClientTest.SendAsync_CorrectVersionSelected_LoopbackServer', Elapsed: 00:14:23
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpClientHandler_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpProtocolTests_Dribble.CustomMethod_SentUppercasedIfKnown', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.PreAuthenticate_AuthenticatedUrl_ThenTryDifferentUrl_SendsAuthHeaderOnlyIfPrefixMatches', Elapsed: 00:14:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests.GetAsync_RequestVersion0X_ThrowsNotSupportedException', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_DiagnosticsTest_Http2.SendAsync_HeadersAreInjectedOnRedirects', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.Http1RawResponseStreamConformanceTests.ReadWrite_Success', Elapsed: 00:14:24
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandler_HttpProtocolTests_Dribble.GetAsyncVersion11_BadResponseVersion_ThrowsOr00', Elapsed: 00:14:07
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandlerTest_Cookies_Http2.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:14:22
[Long Running Test] 'System.Net.Http.Functional.Tests.SyncHttpHandlerTest_Cookies.GetAsync_SetCookieContainer_CookieSent', Elapsed: 00:14:42
[Long Running Test] 'System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.GetAsync_CancelPendingRequests_DoesntCancelReadAsyncOnResponseStream', Elapsed: 00:14:07
...
[EXECUTION TIMED OUT]
```

</details>

All tracked occurrences:
Day | Run | OS | Notes | Runtime
-- | -- | -- | -- | --
**6/30-7/21**
7/9 | PR #71849 | Centos.7.Amd64.Open | 37 tests | CoreCLR
7/5 | PR #71652 | Debian.10.Amd64.Open | 37 tests | CoreCLR
6/30 | PR #71498 | Centos.7.Amd64.Open | 38 tests | CoreCLR
6/30 | PR #71440 | SLES.15.Amd64.Open | 38 tests | CoreCLR
**4/1-4/22** (approx)
4/15 | main | Ubuntu.1804.Amd64.Open |  | CoreCLR
4/3 | main | SLES.15.Amd64.Open |  | CoreCLR
4/12 | PR #67670 | Ubuntu.1804.Amd64.Open | | CoreCLR
4/12 | PR #67746 | Ubuntu.1604.Amd64.Open | | CoreCLR
4/8 | PR #67636 | Ubuntu.1604.Amd64.Open | | CoreCLR
4/6 | PR #67589 | Ubuntu.1604.Amd64.Open | | CoreCLR

|Build|Kind|Start Time|
|---|---|---|
[1732324](https://dev.azure.com/dnceng/public/_build/results?buildId=1732324)|Rolling|2022-22-04|

**Comment by CarnaViire** [comment]
Seems like https://github.com/dotnet/runtime/issues/56477 is still lurking?

**Comment by karelz** [comment]
Triage: We should look at frequency. Unless it happens on regular basis, given that it is not actionable, we would just keep observing.
@rzikm can you please add table like the one in #56477 in top post?

**Comment by wfurt** [comment]
We can add timer to test constructor and call `Environment.FailFast` to produce core. (or use diag client) 
We would need to time it so it kicks in before Helix. 
That would allow us to proceed with investigation.

**Comment by danmoseley** [comment]
I've never seen this before -- it seems these tests are all jammed waiting on some common resource for 15 minutes?

**Comment by wfurt** [comment]
Similar to https://github.com/dotnet/runtime/issues/55642. It was thread pool issue back then. I'm not sure if related. cc: @kouvel just in case.

**Comment by kouvel** [comment]
There was a thread pool issue that was fixed recently in https://github.com/dotnet/runtime/pull/68171. It looks like the build for this run was from Apr 14 and does not include the fix, though not sure if it would be related.

**Comment by jakobbotsch** [comment]
We have seen this several times even after #68171 was merged:

```scala
Jobs
    | where Started > ago(20d) and
            ItemsTimeout > 0 and
            Source == "ci/public/dotnet/runtime/refs/heads/main"
    | join kind=inner WorkItems on JobId
    | where FriendlyName == "System.Net.Http.Functional.Tests" and
            Status == "Timeout"
    | order by Finished desc
    | project Finished, ConsoleUri
```

| Finished                 | ConsoleUri                                                                                                                                                                                                                                                                     |
|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 2022-04-28T22:23:55.586Z | https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-2527bf03bde2444581/System.Net.Http.Functional.Tests/1/console.0742c087.log?helixlogtype=result                                                                                           |
| 2022-04-28T10:11:10.061Z | https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-d43ed85bece149b89c/System.Net.Http.Functional.Tests/1/console.086c28d3.log?helixlogtype=result                                                                                           |
| 2022-04-26T10:20:15.472Z | https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-ee2f14df6402479698/System.Net.Http.Functional.Tests/1/console.47ee4613.log?sv=2019-07-07&se=2022-05-16T09%3A58%3A12Z&sr=c&sp=rl&sig=xvsVGm9hmxWqUgiAjsNH7OzWTSSS6PCb9bNMA2NXtpw%3D       |
| 2022-04-24T09:24:27.398Z | https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-e11e08e5b9c1409997/System.Net.Http.Functional.Tests/1/console.621e6331.log?sv=2019-07-07&se=2022-05-14T08%3A58%3A02Z&sr=c&sp=rl&sig=CKiSzG2fBlInvDEw%2Fx4%2BhDs8mnXYJORgtSCKlST5ATE%3D   |
| 2022-04-22T09:25:52.961Z | https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-a6c6151acf794e10bf/System.Net.Http.Functional.Tests/1/console.9df0afea.log?sv=2019-07-07&se=2022-05-12T08%3A50%3A21Z&sr=c&sp=rl&sig=yQUSceslV1bylR%2FGjUjitMkGnLvsqj7w4zh1VzgC5os%3D     |
| 2022-04-15T09:36:09.853Z | https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-feb37e0b90344a1ab1/System.Net.Http.Functional.Tests/1/console.c35113b8.log?sv=2019-07-07&se=2022-05-05T08%3A55%3A37Z&sr=c&sp=rl&sig=Sfw5ZDDtZfyhjEp7dUGU%2Bpugf%2B5CA%2BXq5sCfIbrKauc%3D |

**Comment by karelz** [comment]
@kouvel it is actionable for you? Or should we ask for crashes when it happens?

**Comment by kouvel** [comment]
I wasn't able to repro it locally last I tried. It would be great if a heap dump could be taken during the hang if that would be possible.

**Comment by karelz** [comment]
Triage: It still happens once per 2 weeks. Moving to 8.0 as it is hard to track down and does not have huge impact on CI.

Frequency: TBD

**Comment by wfurt** [comment]
can you please check the last occurrence @rzikm? The way how this is written does not have enough details (as all the links expired) so it is probably not actionable. If we have no recent failures I would suggest to close this and re-open new issue with logs if this ever fails again.

**Comment by rzikm** [comment]
We don't have any recent hits here. So I am closing this.
